docs/inputs.md

```diff
@@ -184,12 +176,12 @@ The table below contains brief descriptions of the input files required to execu
 | parkactive | Acres of Active Park | 
 | openspaceparkpreserve | Acres of Open Park or Preserve | 
 | beachactive | Acres of Active Beach | 
-| district27 |  | 
+| district27 | Special layer reg employer shuttle service around Sorrento Valley | 
 | milestocoast | Distance (miles) to the nearest coast | 
 | acres | Total acres in the mgra (used in CTM) | 
 | land_acres | Acres of land in the mgra (used in CTM) | 
 | effective_acres | Effective acres in the mgra (used in CTM) | 
-| truckregiontype |  | 
+| truckregiontype | CV model parameter | 
```

Comment:
Please check with @JiaXu1024 about what this is currently used for.

Reply:
> Please check with @JiaXu1024 about what this is currently used for.

Hi Bhargava, I just checked and found the column 'truckregiontype' is not used for any model in ABM3. It was used in ABM1 in TransCAD. I think we can remove this field from the file in the future. The corresponding update will be "\src\asim\configs\common\outputs.yaml"